src/utils/useAtomCallback.ts

```diff
@@ -1,18 +1,21 @@
 import { useCallback, useMemo } from 'react'
 import { atom, useAtom } from 'jotai'
 
-import type { Getter, Setter } from '../core/types'
+import type { Getter, Setter, Scope } from '../core/types'
 
 export function useAtomCallback<Result>(
-  callback: (get: Getter, set: Setter) => Result
+  callback: (get: Getter, set: Setter) => Result,
+  scope?: Scope
 ): () => Promise<Result>
 
 export function useAtomCallback<Result, Arg>(
-  callback: (get: Getter, set: Setter, arg: Arg) => Result
+  callback: (get: Getter, set: Setter, arg: Arg) => Result,
+  scope?: Scope
 ): (arg: Arg) => Promise<Result>
 
 export function useAtomCallback<Result, Arg>(
-  callback: (get: Getter, set: Setter, arg: Arg) => Result
+  callback: (get: Getter, set: Setter, arg: Arg) => Result,
+  scope?: Scope
```

Comment:
This is a bit unfortunate and unexpected, but otherwise this hook is useless with scope.